### DISCUSSION TITLE
Can I contact to repository owner

### DISCUSSION THREAD

**Discussion by SalekurPolas** [community/community#49687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have an organization, and I need a repository name. But the repository name with my organization name is taken, and whoever took the name is not using it, even they don't have any single works with that repo. Can I contact them or get that repo name by contacting anyone like GitHub?

**Comment by Asmitta-01** [comment]
I think it is a better idea to just add a distinction to your repository name. If you really want this name you can try to contact the repository owner with this [repo ](https://github.com/paulirish/github-email).
I never test it but it seems useful to retrieve the user email.

**Comment by jge162** [comment]
I believe after so many years of inactivity you can reach out to support for the name. But, the account has to be inactive. I’ve seen this in a post before here in this community.
